x-pack/test/api_integration/deployment_agnostic/apis/observability/slo/create_slo.ts

```diff
@@ -124,6 +124,41 @@ export default function ({ getService }: DeploymentAgnosticFtrProviderContext) {
       });
     });
 
+    it('creates slos with different tags', async () => {
+      const tags = ['test1', 'test2'];
+      const slo1 = {
+        ...DEFAULT_SLO,
+        tags: [tags[0]],
+        name: 'Test SLO for api integration 1',
+      };
+
+      const slo2 = {
+        ...DEFAULT_SLO,
+        tags: [tags[1]],
+        name: 'Test SLO for api integration 2',
+      };
+
+      await Promise.allSettled([
+        sloApi.create(slo1, adminRoleAuthc),
+        sloApi.create(slo2, adminRoleAuthc),
+      ]);
+
+      const definitions = await sloApi.findDefinitions(adminRoleAuthc, {
+        tags: tags.join(','),
+      });
```

Comment:
The findDefinitions helper could take an array, and then in the helper function, it transforms the array to a csv when calling the API. 
I'm saying that because it's easier to deal with arrays than csv, and the tags only needs to become a csv when we call the API. 

But as this is only for test, consider this a nit